Jalasoft.TeamUp.Resumes.DAL.Test/ResumeSQLRepositoryTests.cs

```diff
@@ -0,0 +1,32 @@
+using Jalasoft.TeamUp.Resumes.DAL.Interfaces;
```

Comment:
we do not need unit tests for the DAL please remove this test

Reply:
I delete file.